{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorUserInfo"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" : "},{"kind":"keyword","text":"Any"},{"kind":"text","text":"] { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/arachne\/arerror\/erroruserinfo-1fik2"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/Arachne\/documentation\/Arachne\/ARError\/errorUserInfo-1fik2","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"CustomNSError.errorUserInfo"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"errorUserInfo","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorUserInfo"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" : "},{"kind":"keyword","text":"Any"},{"kind":"text","text":"]"}],"symbolKind":"property","externalID":"s:10Foundation13CustomNSErrorPAAE13errorUserInfoSDySSypGvp::SYNTHESIZED::s:7Arachne7ARErrorO","extendedModule":"Foundation","modules":[{"name":"Arachne"}]},"hierarchy":{"paths":[["doc:\/\/Arachne\/documentation\/Arachne","doc:\/\/Arachne\/documentation\/Arachne\/ARError","doc:\/\/Arachne\/documentation\/Arachne\/ARError\/CustomNSError-Implementations"]]},"references":{"doc://Arachne/documentation/Arachne/ARError":{"role":"symbol","title":"ARError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ARError"}],"abstract":[{"type":"text","text":"Errors that can be thrown by Arachne networking library"}],"identifier":"doc:\/\/Arachne\/documentation\/Arachne\/ARError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ARError"}],"url":"\/documentation\/arachne\/arerror"},"doc://Arachne/documentation/Arachne/ARError/errorUserInfo-1fik2":{"role":"symbol","title":"errorUserInfo","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorUserInfo"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" : "},{"kind":"keyword","text":"Any"},{"kind":"text","text":"]"}],"abstract":[],"identifier":"doc:\/\/Arachne\/documentation\/Arachne\/ARError\/errorUserInfo-1fik2","kind":"symbol","type":"topic","url":"\/documentation\/arachne\/arerror\/erroruserinfo-1fik2"},"doc://Arachne/documentation/Arachne/ARError/CustomNSError-Implementations":{"role":"collectionGroup","title":"CustomNSError Implementations","abstract":[],"identifier":"doc:\/\/Arachne\/documentation\/Arachne\/ARError\/CustomNSError-Implementations","kind":"article","type":"topic","url":"\/documentation\/arachne\/arerror\/customnserror-implementations"},"doc://Arachne/documentation/Arachne":{"role":"collection","title":"Arachne","abstract":[{"type":"text","text":"Arachne is a lightweight, minimalistic, zero dependencies networking layer for apps using Swift Concurrency (async\/await) developed in Swift, that provides an opinionated abstraction layer to remove boilerplate code."}],"identifier":"doc:\/\/Arachne\/documentation\/Arachne","kind":"symbol","type":"topic","url":"\/documentation\/arachne"}}}